exposed-jodatime/src/test/kotlin/org/jetbrains/exposed/JodaTimeDefaultsTest.kt

```diff
@@ -224,17 +224,10 @@ class JodaTimeDefaultsTest : DatabaseTestsBase() {
 
     @Test
     fun testDefaultExpressions01() {
-        fun abs(value: Int) = object : ExpressionWithColumnType<Int>() {
-            override fun toQueryBuilder(queryBuilder: QueryBuilder) = queryBuilder { append("ABS($value)") }
-
-            override val columnType: IColumnType<Int> = IntegerColumnType()
-        }
-
         val foo = object : IntIdTable("foo") {
             val name = text("name")
             val defaultDateTime = datetime("defaultDateTime").defaultExpression(CurrentDateTime)
             val defaultDate = date("defaultDate").defaultExpression(CurrentDate)
-            val defaultInt = integer("defaultInteger").defaultExpression(abs(-100))
```

Comment:
I will add tests for complex expressions as defaults in a separate pull request.